app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,49 @@
+class CommentsController < ApplicationController
+  before_action :set_comment, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @comments = Comment.all.paginate page: params[:page]
```

Comment:
Not sure what alex's pref is, but I always like to put hash params in parens, I think it looks really weird without them.

method(key: value)

Thoughts?